avo-hq/avo
# feature: tabs

# Description


Fixes https://github.com/avo-hq/avo/issues/813
Fixes https://github.com/avo-hq/avo/issues/627
Fixes https://github.com/avo-hq/avo/issues/623

[Docs](https://docs.avohq.io/2.0/tabs.html#panels)

# Checklist:

- [x] I have made corresponding changes to the [documentation](https://docs.avohq.io/2.0/tabs.html#panels)


## Manual review steps


1. Go to the user resource
2. see the tabs at the bottom
3. click them and see the content loading
4. go to edit
5. click on the tabs to see them working
6. save record
7. see it updates the birthday

Manual reviewer: please leave a comment with output from the test if that's the case.

## Screenshots

![CleanShot 2022-06-15 at 18 38 29](https://user-images.githubusercontent.com/1334409/173868580-1fc6a4e2-b358-4b0f-9443-5de0f502cb1e.gif)
